crates/proof-of-sql/src/base/polynomial/interpolate.rs

```diff
@@ -13,7 +13,14 @@ use num_traits::{Inv, One, Zero};
 ///
 /// For any polynomial, `f(x)`, with degree less than or equal to `d`, we have that:
 /// `f(x) = sum_{i=0}^{d} (-1)^(d-i) * (f(i) / (i! * (d-i)! * (x-i))) * prod_{i=0}^{d} (x-i)`
-/// unless x is one of 0,1,...,d, in which case, f(x) is already known.
+/// unless x is one of 0,1,...,d, in which case, f(x) is already known.\
+/// 
+///  # Panics
```

Comment:
I seems as though this should never panic. So, this is a case where `allow`ing the lint is appropriate. (With a `reason =`.)